### PR TITLE
Adding new function test

### PR DESCRIPTION
**Pull request by mixicek**
Adding new function test

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function test() {
+    console.log("This is awesome!");
+}
+
```
Comment (Ewager1) [inline script.js:4]: Please remove console logs and return items in functions instead.

Comment (mixicek) [reply script.js:4]: @Ewager1, thanks for the comment. I agree and I made the change.

Comment (Ewager1) [reply script.js:4]: @mixicek thanks. This looks like a good fix. I'll consider you for a promotion.

Comment (Ewager1) [reply script.js:4]: Much better. Next time, consider using more descriptive variable names in the future.

**Review by Ewager1 — APPROVED** [review]

**Pull request merged**